package.json

```diff
@@ -30,9 +30,9 @@
     "util.js"
   ],
   "dependencies": {
+    "@digitalbazaar/ed25519-verification-key-2020": "^1.0.0",
     "@digitalbazaar/http-client": "^1.0.0",
     "base64url-universal": "^1.1.0",
-    "crypto-ld": "^3.5.1",
```

Comment:
The changelog says that `crypto-ld@4` is being used, but it looks like it's being dropped entirely here.  Maybe just need to revise the changelog?

Reply:
@mattcollier I think what I was going for was like "uses the newer key suite versions, and _they_ use crypto-ld 4"? But yeah, I'll clarify the changelog.

Reply:
Ok, yeah then I would add a separate point `Remove `crypto-ld` dependency.

Reply:
Reworded, thanks!